tests/sqlparser_common.rs

```diff
@@ -1039,6 +1039,24 @@ fn parse_select_group_by() {
     );
 }
 
+#[test]
+fn parse_select_group_by_grouping_sets() {
+    let sql =
+        "SELECT brand, size, sum(sales) FROM items_sold GROUP BY size, GROUPING SETS ((brand), (size), ())";
+    let select = verified_only_select(sql);
+    assert_eq!(
+        vec![
+            Expr::Identifier(Ident::new("size")),
+            Expr::GroupingSets(vec![
+                vec![Expr::Identifier(Ident::new("brand"))],
+                vec![Expr::Identifier(Ident::new("size"))],
+                vec![],
+            ])
+        ],
+        select.group_by
+    );
+}
```

Comment:
I think we should also add some test here for ROLLUP and CUBE

Reply:
added